R/DBH_metadata.R

```diff
@@ -2,87 +2,119 @@
 #'
 #' @description Gets information on variables included in DBH datasets (type of variable, data type).
 #'
-#' @param table_id A vector of code names for the datasets to get variable information for
-#' @importFrom httr GET
-#' @importFrom httr content
-#' @importFrom readr read_delim
-#' @importFrom readr col_character
-#' @importFrom readr locale
-#' @importFrom readr show_progress
-#' @importFrom dplyr %>%
-#' @importFrom tibble as_tibble
```

Comment:
Vi trenger ikke importere funksjonene til NAMESPACE når vi bruker `::`, som er gjort for alle disse. Synes det er enklest å lese koden bare med `::`. Med mindre funksjonene brukes masse, er det også det [Wickham](http://r-pkgs.had.co.nz/namespace.html#imports) anbefaler: «In fact, this is what I recommend: list the package in DESCRIPTION so that it’s installed, then always refer to it explicitly with pkg::fun(). Unless there is a strong reason not to, it’s better to be explicit.»